scalafmt-core/shared/src/main/scala/org/scalafmt/config/DanglingParentheses.scala

```diff
@@ -36,10 +36,11 @@ object DanglingParentheses {
     case object `class` extends Exclude
     case object `trait` extends Exclude
     case object `enum` extends Exclude
+    case object `extension` extends Exclude
     case object `def` extends Exclude
 
     implicit val reader: ConfCodec[Exclude] =
-      ReaderUtil.oneOf[Exclude](`class`, `trait`, `enum`, `def`)
+      ReaderUtil.oneOf[Exclude](`class`, `trait`, `enum`, `extension`, `def`)
```

Comment:
A corresponding part in the documentation can be updated. In `danglingParentheses.exclude` it's stated that `currently, class, trait and def are supported`. I haven't included there the enum by myself. You can add both extension and enum if you want.

Reply:
Added it!